nrgpy/read/logr.py

```diff
@@ -119,6 +119,47 @@ def set_timestamp_col(self) -> None:
         else:
             self.timestamp_col = "Timestamp"
 
+    def create_data_df_wrapper(self, header_len: int) -> None:
+        self.suffix = str(self.filename).lower().split(".")[-1]
+        if self.suffix == "log":
+            try:
+                self.create_data_df(header_len)
+            except:
+                try:
+                   old_file_name = self.filename
+                   new_file_name = self.filename[:-4] + "_modified.log"
+                   if os.path.exists(new_file_name):
+                       os.remove(new_file_name)
+                   self.filename = new_file_name
+    
+                   file_line_num = 0
+                   expected_number_columns = len(logr_log_columns)
+                   with open(old_file_name) as in_file, open(self.filename, "w") as out_file:
+                       for line in in_file:
+                           if file_line_num > header_len and line.count("\t") >= expected_number_columns:
+                               split_line = line.split("\t")
+                               new_line = ""
+                               for col_count, col_str in enumerate(split_line):
+                                   if col_count < expected_number_columns - 1:
+                                       new_line = new_line + col_str + "\t"
+                                   else:
+                                       new_line = new_line + col_str + " "
+                               new_line = new_line[:-1]
+                           else:
+                               new_line = line
+                           out_file.write(line.replace(line, new_line))
+                           
+                           file_line_num = file_line_num + 1
+                   # Try to create data_df again
+                   self.create_data_df(header_len)
+                except:
```

Comment:
if you know what exceptions are OK it would be better to add those here...

at the least consider catching it as an exception:

```python
except Exception:
    pass
```